### PR TITLE
Moose 5.0.0

### PR DESCRIPTION
**Pull request by Lipt0nas**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+### v6.2.0
+### **Checksummed and Optimized**
+---
+* Update moose tracker to v5.0.0 which introduces automatic context sharing and QoL improvements to development
+* Removed `moose_app_version` field from config
```
Comment (LukasPukenis) [inline changelog.md:5]: Could we add additional note that it's now ignored if present?

**Review by LukasPukenis — DISMISSED** [review]
The diff stats <3 <3 <3

**Review by matszczygiel — APPROVED** [review]
+1

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+### v6.2.0
+### **Checksummed and Optimized**
```
Comment (matszczygiel) [inline changelog.md:2]: Go for `UNRELEASED`

**Pull request closed without merging**